travis.sh

```diff
@@ -18,8 +18,10 @@ BOOST=build/examples/Calc/BoostCalculatorSteps
 if [ -f $GTEST ]; then
     $GTEST >/dev/null &
     cucumber examples/Calc
+    wait
```

Comment:
Without this it's possible for the step definition executable to still be running when cucumber is finished. Keeping the port in use, if timing is working against us just enough then this can result in an inability to open a new listening socket on that port for the BOOST test case below.